### DISCUSSION TITLE
[DEVELOPMENT] How to check if Handlebar is applied to field?

### DISCUSSION THREAD

**Discussion by upgreydd** [Budibase/budibase#5725] [Help]
Hey. I'm curious how to detect if the value is modified by the handlebar, for example in a file:
`packages/bbui/src/Table/BooleanRenderer.svelte` we have `value` and I'd like to render the handlebar if it's in use instead of default styling.
I didn't find any function to retrieve the handlebar from the value level. Any tips, please?

**Comment by melohagan** [comment]
Hey @Upgreydd 

I imagine the *BooleanRenderer* should be agnostic to how the value was generated. 

The *CellRenderer* may be a better place to check, although I'm not overly active in the codebase myself. There's a `getCellValue` function, and within it I think the `processStringSync` function is handling the handlebars.
